### PR TITLE
started workshop blog post

### PR DESCRIPTION
**Pull request by richardjgowers**

### CODE REVIEW

File: _posts/2018-08-30-numfocus-workshop.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: NumFocus Sponsored Workshop and Hackathon
```
Comment (orbeckst) [inline _posts/2018-08-30-numfocus-workshop.md]: NumFOCUS

File: _posts/2018-08-30-numfocus-workshop.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+We are happy to announce that thanks to a [NumFOCUS development grant][numfocus]
+we will be hosting a 2 day workshop and hackathon aimed at introducing researchers to MDAnalysis.
+The event will be free to attend and will be held at Northwestern Universty, Evanston IL on the 12th and 13th of November.
```
Comment (orbeckst) [inline _posts/2018-08-30-numfocus-workshop.md]: University

Comment (orbeckst) [reply _posts/2018-08-30-numfocus-workshop.md]: 2018

File: _posts/2018-08-30-numfocus-workshop.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+In addition, there are travel grants available to attendees aimed at promoting diversity in STEM.
+
+The first day of the workshop will focus on covering the basics of the library and how to apply
+this to your own research.
+The second day of the workshop will be designed to help attendees
+to start contributing to MDAnalysis
+as well as how to build production ready analysis code.
```
Comment (orbeckst) [inline _posts/2018-08-30-numfocus-workshop.md]: production-ready

File: _posts/2018-08-30-numfocus-workshop.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: NumFOCUS Sponsored Workshop and Hackathon
+---
+
+We are happy to announce that thanks to a [NumFOCUS development grant][numfocus]
```
Comment (micaela-matta) [inline _posts/2018-08-30-numfocus-workshop.md]: add a nice image here too

File: _posts/2018-08-30-numfocus-workshop.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+we will be hosting a 2 day workshop and hackathon aimed at introducing researchers to MDAnalysis.
+The event will be free to attend and will be held at Northwestern University, Evanston IL on the 12th and 13th of November 2018.
+In addition, there are travel grants available to attendees aimed at promoting diversity in STEM.
+
+The first day of the workshop will focus on covering the basics of the library and how to apply
+this to your own research.
+The second day of the workshop will be designed to help attendees
```
Comment (micaela-matta) [inline _posts/2018-08-30-numfocus-workshop.md]: The second day of the workshop will introduce git and GitHub and help you start contributing to MDAnalysis with your own analysis code.

File: _posts/2018-08-30-numfocus-workshop.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: NumFOCUS Sponsored Workshop and Hackathon
+---
+
+We are happy to announce that thanks to a [NumFOCUS development grant][numfocus]
+we will be hosting a 2 day workshop and hackathon aimed at introducing researchers to MDAnalysis.
+The event will be free to attend and will be held at Northwestern University, Evanston IL on the 12th and 13th of November 2018.
+In addition, there are travel grants available to attendees aimed at promoting diversity in STEM.
```
Comment (micaela-matta) [inline _posts/2018-08-30-numfocus-workshop.md]: mention that the travel grants are meant to encourage people from outside Chicago to participate and women and minorities are especially encouraged to apply

**Pull request merged**